docs/product-business-resource-planner-guide/src/main/asciidoc/optimizer-download-proc.adoc

```diff
@@ -0,0 +1,23 @@
+[id='optimizer-download-proc']
+= Download {PLANNER}
+
+{PLANNER} is production ready.
+The API is stable but backward incompatible changes can occur.
+
+.Prerequisite
+The {PRODUCT} has been downloaded and installed from the https://access.redhat.com[Red Hat Customer Portal]. See Installing Decision Manager for more information about installing {PRODUCT}.
```

Comment:
Would you be giving the exact location  for downloads later?

Reply:
Included the Enterprise URL attribute.